sources/packages/api/src/services/institution-location/institution-location.service.ts

```diff
@@ -56,6 +57,57 @@ export class InstitutionLocationService extends RecordDataModelService<Instituti
     return this.repo.save(saveLocation);
   }
 
+  /**
+   * Updating institution location.
+   * @param data Payload of updated data.
+   * @param locationId Location Id to update.
+   * @returns Updated location as InstitutionLocation.
+   */
+  async updateLocationPrimaryContact(
+    data: InstitutionLocationPrimaryContactModel,
+    locationId: number,
+  ): Promise<InstitutionLocation> {
+    const saveLocation = {
+      primaryContact: {
+        firstName: data.primaryContactFirstName,
+        lastName: data.primaryContactLastName,
+        email: data.primaryContactEmail,
+        phone: data.primaryContactPhone,
+      },
+      id: locationId,
+    };
+
+    return this.repo.save(saveLocation);
+  }
+
+  /**
+   * Updating institution location.
+   * @param data Payload of updated data.
+   * @param locationId Location Id to update.
+   * @returns Updated location as InstitutionLocation.
+   */
+  async updateLocationForAEST(
```

Comment:
We can avoid client type names in service method. Service methods are independent of client types.

Reply:
Yeah but in this case we use it only for the AEST user, I can rename it to say update location. Just to make it generic